templates/admin/job_template/index.html.ep

```diff
@@ -1,15 +1,99 @@
+                        </div>
+                    </div>
+                    <div class="form-group">
+                        <label for="editor-keep-important-logs-in-days" class="col-sm-2 control-label">Keep important logs for</label>
+                        <div class="col-sm-10">
+                            <input type="number" min="0" class="form-control" id="editor-keep-important-logs-in-days" name="keep_important_logs_in_days" value="<%= $group->keep_important_logs_in_days %>"> days
+                        </div>
+                    </div>
+                    <div class="form-group">
+                        <label for="editor-keep-results-in-days" class="col-sm-2 control-label">Keep results for</label>
+                        <div class="col-sm-10">
+                            <input type="number" min="0" class="form-control" id="editor-keep-results-in-days" name="keep_results_in_days" value="<%= $group->keep_results_in_days %>"> days
+                        </div>
+                    </div>
+                    <div class="form-group">
+                        <label for="editor-keep-important-results-in-days" class="col-sm-2 control-label">Keep important results for</label>
+                        <div class="col-sm-10">
+                            <input type="number" min="0" class="form-control" id="editor-keep-important-results-in-days" name="keep_important_results_in_days" value="<%= $group->keep_important_results_in_days %>"> days
+                        </div>
+                    </div>
+                    <div class="form-group">
+                        <label for="editor-default-priority" class="col-sm-2 control-label">Default priority for jobs</label>
+                        <div class="col-sm-10">
+                            <input type="number" class="form-control" id="editor-default-priority" name="default_priority" value="<%= $group->default_priority %>">
+                        </div>
+                    </div>
+                    <div class="form-group">
+                        <label for="editor-default-priority" class="col-sm-2 control-label">Description</label>
+                        <div class="col-sm-10">
+                            <textarea class="form-control" id="editor-description" name="description" ><%= $group->description %></textarea>
+                        </div>
+                    </div>
+                    <div class="form-group">
+                        <label for="editor-info" class="col-sm-2 control-label"><span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span></label>
+                        <div class="col-sm-10">
+                            <span id="editor-info">All time-related properties (measured in days) can be set to <em>0</em> to denote infinity.</span>
+                        </div>
+                    </div>
```

Comment:
All the blocks in before look pretty repetitive. I wonder if there is a mojo shortcut